frontend/src/metabase/pulse/components/PulseEdit.jsx

```diff
@@ -123,6 +127,12 @@ export default class PulseEdit extends Component {
     const { pulse, formInput } = this.props;
     const isValid = pulseIsValid(pulse, formInput.channels);
     const attachmentsEnabled = emailIsEnabled(pulse);
+    const link = (
+      <a
+        className="link"
+        href={MetabaseSettings.docsUrl("users-guide/07-dashboards")}
```

Comment:
I'm surprised `MetabaseSettings` is where we have the `docsUrl` method. Feels like that should come from `metabase/lib/urls` (Not something you did just noticing it here).

Reply:
I wonder if it's so we can send you to the docs matching your instance version?

Reply:
Yeah I bet it is, just feels weird that it's there vs in the URLs lib where most other routing related methods are.